package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "human-signals",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
There is a Gulp task [`gulp releaseMinor`](https://github.com/ehmicky/dev-tasks#gulp-releasepatch-gulp-releaseminor-gulp-releasemajor) that increments package version, create a GitHub release, waits for CI to succeed then publishes to npm. This is done on `master`, i.e. outside of PRs.
Therefore, would it be possible to remove the `version` change (also in `package-lock.json`)?

Reply:
Fixed. Should I leave my update to `CHANGELOG.md`?

Reply:
Yes it's good :+1:. Thanks!